packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -426,8 +438,10 @@ export class Conductor extends EventEmitter<ConductorEvents> {
 				return this.getCurrentTime()
 			}
 
+			let newDevicePs: Promise<DeviceContainer<DeviceOptionsBase<any>> | undefined>
```

Comment:
Could we try not to use  abbreviations? I am guessing that `Ps` here means `Promise`, but for me `Ps` means `Playstation` :P

Reply:
Goes for all your `Ps` naming :)

Reply:
I renamed it